### PR TITLE
patched SDK_LOCAL client libs.  updated BLAST to 2.8.1.  passing all unit tests

### PR DESCRIPTION
**Pull request by dcchivian**

### CODE REVIEW

File: lib/kb_blast/kb_blastImpl.py [hunk 1]
```diff
@@ -19,14 +19,15 @@
 from Bio.SeqRecord import SeqRecord
 from Bio.Alphabet import generic_protein
 #from biokbase.workspace.client import Workspace as workspaceService
-from Workspace.WorkspaceClient import Workspace as workspaceService
+#from Workspace.WorkspaceClient import Workspace as workspaceService
```
Comment (bio-boris) [inline lib/kb_blast/kb_blastImpl.py]: You can delete these unused imports

File: test/kb_blast_server_test.py [hunk 2]
```diff
@@ -14,7 +14,9 @@
 
 from pprint import pprint  # noqa: F401
 
-from biokbase.workspace.client import Workspace as workspaceService
+#from biokbase.workspace.client import Workspace as workspaceService
+#from Workspace.WorkspaceClient import Workspace as workspaceService
```
Comment (bio-boris) [inline test/kb_blast_server_test.py]: you can remove the comments

**Review by bio-boris — APPROVED** [review]

File: lib/kb_blast/kb_blastImpl.py (hunk 1 above)
Comment (dcchivian) [reply lib/kb_blast/kb_blastImpl.py]: sure.  I tend to accumulate cruft.  comes from being old and learning how to code before version control systems took over.

Comment (dcchivian) [reply lib/kb_blast/kb_blastImpl.py]: removed comment lines

File: test/kb_blast_server_test.py (hunk 2 above)
Comment (dcchivian) [reply test/kb_blast_server_test.py]: removed comment lines

**Pull request merged**